picard/ui/__init__.py

```diff
@@ -113,9 +118,18 @@ def __init__(self, parent=None):
     def keyPressEvent(self, event):
         if event.matches(QtGui.QKeySequence.Close):
             self.close()
+        elif event.matches(QtGui.QKeySequence.HelpContents) and self.help_url:
+            self.show_help()
         else:
             super().keyPressEvent(event)
 
+    def show_help(self):
+        if self.help_url:
+            url = self.help_url
+            if url.startswith('/'):
```

Comment:
I wonder if we still need this.

Reply:
yes, for links from option pages. They define only partial URLs (to avoid having to use DOCS_BASE_URL everywhere)